### PR TITLE
Feature/dbilokha/8 fix build

### PR DESCRIPTION
**Pull request by danbilokha**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     "typescript": "^3.8.3",
     "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11",
+    "webpack-node-externals": "^1.7.2",
     "winston": "^3.2.1",
```
Comment (Proladge) [inline package.json:91]: @danbilokha Just curious is it ok to have in dev deps those pakages we will need in prod?

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ],
+  "include": [
+    "server/**/*.ts"
+  ],
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "dist",
+    "rootDir": "server",
+    "sourceMap": true,
```
Comment (Proladge) [inline tsconfig.json:13]: @danbilokha Do we need source map in production version?
I suppose this is the only one situation when are going to build app, right?

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const nodeExternals = require('webpack-node-externals');
```
Comment (Proladge) [inline webpack.config.js:1]: Require is fine here, right?

File: package.json (hunk 1 above)
Comment (danbilokha) [reply package.json:91]: dev deps will not be present on prod (at least I think so, it's the way I expect it to work)

File: tsconfig.json (hunk 2 above)
Comment (danbilokha) [reply tsconfig.json:13]: not sure :D

File: webpack.config.js (hunk 3 above)
Comment (danbilokha) [reply webpack.config.js:1]: i think so as it's in webpack config itself

**Comment by danbilokha** [comment]
approved by @Proladge

**Pull request merged**